### ISSUE TITLE
problem installing ngrx/entity from the nightly builds (npm)

### ISSUE THREAD

**Issue by walrus95** [ngrx/platform#485]
I'm having a problem installing ngrx/entity from the nightly builds:

```
$ ng new foo
$ cd foo
$ npm install github:ngrx/entity-builds --save
(node:10383) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: ENOENT: no such file or directory, stat '/home/walrus95/.npm/_cacache/tmp/git-clone-cb681789/ngrx/entity.es5.js'
(node:10383) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
npm ERR! cb() never called!

npm ERR! This is an error with npm itself. Please report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/walrus95/.npm/_logs/2017-10-15T22_41_42_580Z-debug.log
$ node -v
v8.4.0
$ npm -v
5.4.2
```

Thank you!

**Comment by brandonroberts** [comment]
This error message says this is an npm issue and it works correctly with yarn. I'll look further into it this week. In the meantime you can use yarn to install

**Comment by brandonroberts** [comment]
@ngrx/entity has been officially released to npm as `4.1.0`, so you can use the official npm channel to install. We'll continue to monitor this issue.

**Comment by bmayen** [comment]
I get:
npm i @ngrx/entity
npm ERR! code E404
npm ERR! 404 Not Found: @ngrx/entity@^4.1.0
